web/frontend/src/components/slam/index.svelte

```diff
@@ -351,12 +351,13 @@ const clearRefresh = () => {
   clear2dRefresh?.();
 };
 
-const handleEndMapping = () => {
+const handleEndMapping = async () => {
   hasActiveSession = false;
   mappingSessionEnded = true;
   clearRefresh();
   clearInterval(durationInterval);
-  overrides?.endMappingSession(sessionId);
+  await overrides?.endMappingSession(sessionId);
+  notify.success("Mapping session complete", "You can find the newly generated map in the location's SLAM library.")
```

Comment:
Should we alert the user of a failure in that event as well?
```suggestion
  try {
    await overrides?.endMappingSession(sessionId);
    notify.success(
      "Mapping session complete",
      "You can find the newly generated map in the location's SLAM library."
    )
  } catch {
    notify.danger('Some title', 'Some message')
  }
  
```

Reply:
Sorry for wasting your time, I just realized that the fail notify is actually already happening from app so I will put my success notify there as well. Closing this PR